packages/dd-trace/src/appsec/iast/telemetry/index.js

```diff
@@ -5,11 +5,19 @@ const telemetryLogs = require('./log')
 const { Verbosity, getVerbosity } = require('./verbosity')
 const { initRequestNamespace, finalizeRequestNamespace, globalNamespace } = require('./namespaces')
 
+function isMetricsEnabled (metrics) {
+  return process.env.DD_TELEMETRY_METRICS_ENABLED !== undefined ? metrics : true
+}
```

Comment:
I understand what this does but let's at least add a comment here to say that this should ultimately be done in config.js instead ?

Reply:
This is related only with IAST metrics, and `DD_TELEMETRY_METRICS_ENABLED` is general configuration field, if `DD_TELEMETRY_METRICS_ENABLED` is undefined the general status of the property should be `false`, and only true for this file.

I think that we could change the method name to something like `isIastMetricsEnabled` to avoid misunderstandings

Reply:
for the record: what I mean is the config value should be undefined in the first place from config.js. Instead we should directly check `config.telemetry.enabled == null ? true : config.telemetry.enabled`